### PR TITLE
Feature/query multiple tables

### PR DESCRIPTION
**Pull request by thasipablo**
**Requirements:**

In this project, I added new tables and foreign key columns to the existing animals' table to model one-to-many relationships. Afterward, I used JOIN to query the data.

**Interaction Screenshots:**
Screen Shot 2023-09-30 at 4 13 01 PM
Screen Shot 2023-09-30 at 4 44 44 PM
Screen Shot 2023-09-30 at 4 45 00 PM
Screen Shot 2023-09-30 at 5 29 39 PM
Screen Shot 2023-09-30 at 12 37 27 PM
Screen Shot 2023-09-30 at 6 06 03 PM

### CODE REVIEW

File: queries.sql [hunk 1]
```diff
@@ -33,3 +33,42 @@ SELECT AVG(weight_kg) FROM animals;
...
+SELECT s.name AS species_name, COUNT(*) AS animal_count
+  FROM animals AS a
+  INNER JOIN species AS s ON a.species_id = s.id
+  GROUP BY s.name;
+
+SELECT a.name
+  FROM animals AS a
+  INNER JOIN species AS s ON a.species_id = s.id
+  INNER JOIN owners AS o ON a.owner_id = o.id
+  WHERE o.full_name = 'Jennifer Orwell' AND s.name = 'Digimon';
```
Comment (mikethreels) [inline queries.sql:61]: - [ ] as far as I can tell there is no print screen for this query in your PR message.

**Review by mikethreels — CHANGES_REQUESTED** [review]
Hi Pablo, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- good job creating the queries.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by VICTOR-CHUKWU — APPROVED** [review]
Hi @thasipablo , 
# STATUS: APPROVED! 🥇 💯 
done

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by thasipablo** [comment]
Thank you

**Pull request merged**